src/components/hooks/useFilters.js

```diff
@@ -163,24 +183,30 @@ function useFilters() {
    */
   useEffect(() => {
     if (resetOnQueryChange) {
-      // set locale object
-      locale = _getQuery();
-      // update locale state (swr)
-      _setFilters(locale);
+      // check if current value differs from previous
+      if (prevQuery && prevQuery !== curQuery) {
```

Comment:
er det mon nok bare at sige if(prevQuery)? 
prevQuery og curQuery bør altid være forskellige?

Reply:
jeg må teste det, logikken fejler lidt for mig idag. :D